.eslintrc.json

```diff
@@ -22,10 +25,32 @@
   ],
 
   "rules": {
-    "comma-dangle": ["error", "only-multiline"],
-    "indent": ["error", 2, { "SwitchCase": 1 }],
-    "linebreak-style": ["error", "unix"],
-    "quotes": ["error", "single"],
-    "semi": ["error", "always"]
+    "indent": [
+      2, 
+      4
+    ],
+    "comma-dangle": [
+        "error", 
+        "never"
+    ],
+    "no-unused-vars": [
```

Comment:
We should as a principal never have any unused vars, why make this rule custom? Why not use the default setting?